cli.js

```diff
@@ -15,10 +15,12 @@ const defaults = {
     h: 'help',
     v: 'version',
     l: 'language',
-    o: 'open'
+    o: 'open',
+    i: 'idiom'
```

Comment:
@fernahh What do you think of replacing `idiom` by `locale`? I think it makes it more clear for the users.